example/spec/parallel/syncGranule/SyncGranuleDuplicateHandlingSpec.js

```diff
@@ -58,6 +59,12 @@ describe('When the Sync Granule workflow is configured', () => {
   let pdrModel;
   let testSuffix;
   let testDataFolder;
+  let syncGranuleExecutionArn;
```

Comment:
If these are all top-scoped, consider an array  to iterate over instead.     

That said,I think we should  locally scope this if cleanup in a sub-test afterAll is where these are being used, and make the names locally meaningful.

Reply:
Yup, that's cleaner. Could also put all of these at the top-level in `afterAll` but I think it might be cleaner to leave them. This is a test that we could look at when we get a chance. It's creating a _lot_ of executions.

https://github.com/nasa/cumulus/pull/2252/commits/4ea600e788c2f7a2b16c785c26ed785620525611